doc/source/cluster/kubernetes/user-guides/kuberay-gcs-ft.md

```diff
@@ -308,7 +308,21 @@ Refer to [this section](kuberay-external-storage-namespace-example) in the earli
 (kuberay-redis-cleanup-gate)=
 ### 4. Turn off Redis cleanup
 
-* `ENABLE_GCS_FT_REDIS_CLEANUP`: The feature gate `ENABLE_GCS_FT_REDIS_CLEANUP` is true by default, and users can turn if off by setting the environment variable in [KubeRay operator's Helm chart](https://github.com/ray-project/kuberay/blob/master/helm-chart/kuberay-operator/values.yaml).
+* `ENABLE_GCS_FT_REDIS_CLEANUP`: True by default. You can turn this feature off by setting the environment variable in the [KubeRay operator's Helm chart](https://github.com/ray-project/kuberay/blob/master/helm-chart/kuberay-operator/values.yaml).
+
+```{admonition} Key eviction setup on Redis
+If you disable `ENABLE_GCS_FT_REDIS_CLEANUP` but want Redis to remove GCS metadata automatically,
+set these two options on Redis:
+
+* `maxmemory=<your_memory_limit>`
+* `maxmemory-policy=allkeys-lru`
+
```

Comment:
Nit: Where exactly do you write these options? (Is it `redis.conf`?) It might be good to be totally explicit, for beginners.

Reply:
* You can store the configurations in a ConfigMap and then start the Redis server using that config file ([example](https://github.com/ray-project/kuberay/blob/3754d34ef61734b569d235f5512c27c3f40d82df/ray-operator/config/samples/ray-cluster.external-redis.yaml#L1-L15)).
* You can also directly specify the options in the container command. See https://github.com/ray-project/ray/pull/40949#issuecomment-1799057691 as an example.

Reply:
Maybe @rueian can add a link to https://github.com/ray-project/ray/pull/40949#issuecomment-1799057691 in the PR?

Reply:
Thanks for the info! I think this is important to include then. (I think a nontrivial fraction of users may get stuck/confused without this info)